codec-stomp/src/main/java/io/netty/handler/codec/stomp/StompSubframeEncoder.java

```diff
@@ -28,16 +29,18 @@
  * Encodes a {@link StompFrame} or a {@link StompSubframe} into a {@link ByteBuf}.
  */
 public class StompSubframeEncoder extends MessageToMessageEncoder<StompSubframe> {
+    private static final ByteBuf EOF =
+            Unpooled.unreleasableBuffer(Unpooled.wrappedBuffer(new byte[] { StompConstants.NUL }));
 
     @Override
     protected void encode(ChannelHandlerContext ctx, StompSubframe msg, List<Object> out) throws Exception {
         if (msg instanceof StompFrame) {
             StompFrame frame = (StompFrame) msg;
             ByteBuf frameBuf = encodeFrame(frame, ctx);
             if (frame.content().isReadable()) {
-                out.add(frameBuf);
                 ByteBuf contentBuf = encodeContent(frame, ctx);
-                out.add(contentBuf);
+                ByteBuf buf = Unpooled.wrappedBuffer(frameBuf, contentBuf);
```

Comment:
Why you think this a good change ?

Reply:
@normanmaurer In my project stomp over websocket. If `StompSubframEncodeer` return two msg, I have to aggregate the two msg into one `WebSocketTextFrame` in my `StompOverWebsocketAggregator`. If it return only one msg, I just wrap the msg with `WebSocketTextFrame`.

ps: In my project write DefaultStompFrame only.


My StompOverWebsocketAggregator
```
    @Override
    protected void encode(ChannelHandlerContext ctx, ByteBuf msg, List<Object> out) {
        if (msg == null)
            return;
        out.add(new TextWebSocketFrame(msg.retain()));
    }
```